electives/18741.md

```diff
@@ -0,0 +1,52 @@
+
+## Quiz
+    There are bi-weekly quizzes in the course which amounts to 5 quizzes in total. The quizzes are meant to test concepts covered in recent lectures and not designed to be super hard. They can be
+    taken and finished in a couple of hours depending on familiarity with the material, however usually a few days are provided for the quiz to be completed and submitted.
+
+## Projects
+
+The projects are quite interesting, thought provoking and impart tons of learning in the course of completing them. The best part is that they are well aligned with the pace of the course and cover concepts
+that have been discussed in detail in class. There are four projects in total, each covering a different layer of the network stack. The first project involves writing a Wifi receiver with the transmitter code and algorithm provided. Students will learn how bits are actually modulated and encoded to provide noise immunity by writing their own Viterbi decoder from sratch. The second project focuses on the network layer
+with students implementing their own routing algorithms to get an idea of large scale routing design and challenges. The third project focuses on the transport layer and involves the implementation of delivering a byte stream reliably in the presence of packet loss, re-ordering and duplication. The final project involves designing a content delivery server capable of handling hundereds of concurrent clients. All the projects involve heavy socket programming and dealing with concurrent programs making them challenging and fun at the same time.
+
+## Exams
+
+Exams are not designed to be super tricky or difficult but instead focus on the concepts covered in lectures and in projects. Going through past year paper should help in gaining familiarity with the format,
+number of questions and time management. The exams usually do not test concepts based on memory and a cheatsheet is allowed in order to avoid such memorization and promote understanding the reasons and idea behind concepts well. A few questions can be tricky, but going through the lectures and projects thoroughly should allow one to answer them intuitively.
+
+## How to do well
```

Comment:
any links to pass course websites / exams?

Reply:
Unfortunately no :( Those are released by the professor on canvas